packages/cypress/src/add-commands/login.ts

```diff
@@ -166,11 +179,15 @@ function loginByForm(commandOptions: CommandLoginOptions) {
 const isCustomView = (commandOptions: CommandLoginOptions) =>
   commandOptions.entryPointUriPath === CUSTOM_VIEW_HOST_ENTRY_POINT_URI_PATH;
 
-function loginByOidc(commandOptions: CommandLoginOptions) {
+function loginByOidc(
+  commandOptions: CommandLoginOptions &
+    LoginToMerchantCenterForCustomViewCommandLoginOptions
+) {
```

Comment:
@emmenko Thank you for the feedback. All changes were applied in 153c06cc138d0f4ef9db5a9cf9adca5e3dba7237.
I wonder if this intersection type here works for you, as I didn't want to go for discriminated unions